git-clang-format-14

```diff
@@ -593,4 +628,4 @@ def convert_string(bytes_input):
         return str(bytes_input)
 
 if __name__ == '__main__':
-  main()
+  main()
```

Comment:
Can you please re-add the newline at end of file and set the mode to 755 (we run `chmod +x` in the Dockerfile, but this should be executable in git anyway)?